alfred-api-integrationtests-server/src/main/java/eu/xenit/alfred/api/util/SolrTestHelperImpl.java

```diff
@@ -50,10 +49,15 @@ public SolrTestHelperImpl(
      */
     @Override
     public boolean areTransactionsSynced() {
-        long alfTransaction = alfrescoServerInfo.getAlfTransactionIdDAO();
-        long solrTransaction = solrAdminClient.getLastTransactionId();
-        logger.debug("alf transaction: {}, solr transaction: {}", alfTransaction, solrTransaction);
-        return alfTransaction <= solrTransaction;
+        try {
+            long alfTransaction = alfrescoServerInfo.getAlfTransactionIdDAO();
+            long solrTransaction = solrAdminClient.getLastTransactionId();
+            logger.debug("alf transaction: {}, solr transaction: {}", alfTransaction, solrTransaction);
+            return alfTransaction <= solrTransaction;
```

Comment:
Just. out of curiosity, since this is very disorienting.

should the statement not be alfTransaction == solrTransaction

alf transaction could never be lower than the solr transaction? because the solr transaction is matched on the  alfresco transaction? Unless alfresco was rolled back and it's in some kind of weird bizarre corrupt state?

It's a very confusing statement, but yes, it should work

Reply:
I only indented this, so if you don't mind I will keep changes it out of scope to prevent regression.